src/js/media/html5.js

```diff
@@ -76,7 +76,7 @@ vjs.Html5 = vjs.MediaTechController.extend({
     // In Chrome (15), if you have autoplay + a poster + no controls, the video gets hidden (but audio plays)
     // This fixes both issues. Need to wait for API, so it updates displays correctly
     player.ready(function(){
-      if (this.tag && this.options_['autoplay'] && this.paused()) {
+      if (this.src && this.tag && this.options_['autoplay'] && this.paused()) {
```

Comment:
`this.src` is a function (getter/setter) not a property, so this should be `this.src()`